server/src/Controllers/newsController.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from 'express';
+import newsLibrary from '../Libraries/newsLibrary';
+
+
+
+export const addNews = async (req: Request, res: Response): Promise<void> => {
+    try {
+        const {user_id , news_content} = req.body;
+        console.log("Request Body:", user_id , news_content);
+        await newsLibrary.addNewsForUser(user_id , news_content);
+        console.log("News added successfully");
+        res.status(200).json({ message: "News added successfully" });
```

Comment:
create a constant file for all messages (success and error)

Reply:
also use NEWS_ADDED_SUCCESSFULL or use numbers for success / error messages

Reply:
DONE ✅